client/payment-details/summary/index.tsx

```diff
@@ -113,10 +113,10 @@ const PaymentDetailsSummary = ( {
 		featureFlags: { isAuthAndCaptureEnabled },
 	} = useContext( WCPaySettingsContext );
 
-	// We should only fetch the authorization data if the payment is marked for manual capture
+	// We should only fetch the authorization data if the payment is marked for manual capture and it is not already captured.
 	const shouldFetchAuthorization =
-		charge.amount !== charge.amount_captured &&
-		charge.amount_refunded === 0 &&
+		'captured' in charge && // captured key is only present for manual capture payments. See https://stripe.com/docs/api/charges/object#charge_object-captured
+		! charge.captured &&
```

Comment:
We lost the check for the refunded amount here, why so?

Reply:
I was mistaken and thought that checking for `captured` would be enough. Added it back.

Reply:
Done here [c5b2e48](https://github.com/Automattic/woocommerce-payments/pull/5232/commits/c5b2e483f969fcfbce322c23edea33a2ebdd86d0)